src/pkg/cluster/state.go

```diff
@@ -210,13 +210,14 @@ func (c *Cluster) InitZarfState(ctx context.Context, initOptions types.ZarfInitO
 
 // LoadZarfState returns the current zarf/zarf-state secret data or an empty ZarfState.
 func (c *Cluster) LoadZarfState(ctx context.Context) (state *types.ZarfState, err error) {
+	stateErr := errors.New("failed to load the Zarf State from the cluster, has Zarf been initiated?")
 	secret, err := c.Clientset.CoreV1().Secrets(ZarfNamespaceName).Get(ctx, ZarfStateSecretName, metav1.GetOptions{})
 	if err != nil {
-		return nil, fmt.Errorf("%w. %s", err, message.ColorWrap("Did you remember to zarf init?", color.Bold))
+		return nil, errors.Join(stateErr, err)
```

Comment:
nit but I would still use fmt.Errorf. Newlines look a bit awkward when we print to the user and I think it reads better for the calls to action to be at the end of the error. 
![image](https://github.com/user-attachments/assets/aabcaf38-10dd-4e63-b32c-c97cb09aee57)

Reply:
Will do that for now and look back at formatting errors in the future.